dynamic-simulation/dynamic-simulation-api/pom.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2016, All partners of the iTesla project (http://www.itesla-project.eu/consortium)
```

Comment:
Fix this copyright information. Use `Copyright (c) 2019, RTE (http://www.rte-france.com)`